server/controller/controller.js

```diff
@@ -0,0 +1,96 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+const User = require('../config/sequelize');
+const hashPassword = require('../config/hashPassword');
+const totp = require('../config/totp');
+
```

Comment:
Better store all these values in a file and import from there, so that if we change at one place we don't need to change it across all the files.

Reply:
This files are imported only once. So that's why I didn't created another file to import all this file.

Reply:
Yes, but it is a good policy to do this. When the codebase gets larger, it helps make it easy to work it.